TA_dataset/lib/dataset_query_api_client/api/default/post_queries.py

```diff
@@ -107,11 +107,10 @@ def sync_detailed(
         Response[QueryResult]
     """
 
-    kwargs = _get_kwargs(
-        client=client,
-        json_body=json_body,
+    kwargs = _get_kwargs(client=client, json_body=json_body)
+    logger.info(
```

Comment:
Can't it be an issue to call any func on the arg, that has a default value `None`?

Reply:
Also do we need `info` log or `debug` is enough?

Reply:
it wont create the issue. this is the way which the application is written from the scratch, BTW i didnt made any updated to this method. Just added a logger for future needs. And moreover converted the info to debug as mentioned